iron-scroll-manager.html

```diff
@@ -56,6 +56,7 @@
       /**
        * The current element that defines the DOM boundaries of the
        * scroll lock. This is always the most recently locking element.
+       * @return {!Node}
```

Comment:
`currentLockingElement` can be undefined - this was wrongly annotated previously as it was saying that `currentLockingElement` can be null

Reply:
Done.